Mathlib/Computability/Language.lean

```diff
@@ -85,6 +85,11 @@ instance : Add (Language α) :=
 instance : Mul (Language α) :=
   ⟨image2 (· ++ ·)⟩
 
+/-- The intersection of two languages `l` and `m` is the language made of the
+strings `x` where `x ∈ l` and `x ∈ m`. -/
+instance instInter : Inter (Language α) :=
+  ⟨((· ∩ ·) : Set (List α) → Set (List α) → Set (List α))⟩
+
```

Comment:
Other instances don't have names. Does this one need a name?

Reply:
Fair point. Admittedly I personally don't like the implicit naming, but consistency is good.

Reply:
I mean, when would you need to explicitly refer to the instance, anyway?

Reply:
In principle if I am trying to unfold a specific instance's implementation I prefer if it's not the generated name, but this probably doesn't come up too often...

Reply:
Fixed btw.

Reply:
> In principle if I am trying to unfold a specific instance's implementation I prefer if it's not the generated name, but this probably doesn't come up too often...

Looks like the preferred solution is to have something like e.g. `Language.add_def`.